e2e/playwright/fixtures/editorFixture.ts

```diff
@@ -31,19 +32,32 @@ export class EditorFixture {
 
   private _expectEditorToContain =
     (not = false) =>
-    (
+    async (
       code: string,
       {
         shouldNormalise = false,
         timeout = 5_000,
       }: { shouldNormalise?: boolean; timeout?: number } = {}
     ) => {
+      const wasPaneOpen = await this.isPaneOpen()
```

Comment:
I updated to `checkIfPanIsOpen`, I didn't do the `paneState` thing this go because there are other notions of "state" in this class already, but we'll include something similar when we add a "serialise" function to the fixture as we need it.